assets/components/announcement.scss

```
@@ -1,9 +1,9 @@
 .announcement {
     .header-announcement {
         background: #64005A;
-        /*
         .banner::after {
-                content: "NON-SERVICE DISRUPTING MAINTENANCE: On June 8, 2023, between 09:00 CDT and 12:00 CDT, docs.rackspace.com will be redirected to our new Website.";
+                /* This is a temporary maintenance banner for migration release */
+                content: "SERVICE DISRUPTING MAINTENANCE: On June 29, 2023 between 17:00 CDT and 21:00 CDT all micro sites under the docs.rackspace.com, and the Technical Blog at docs.rackspace.com/blog will be unavailable.";
```

Updated text based on discussions with Kim Headen.
```suggestion
                content: "SERVICE DISRUPTING MAINTENANCE: On June 29, 2023 between 17:00 CDT and 21:00 CDT docs.rackspace.com and docs.rackspace.com/blog will be unavailable while they are redirected to a new website.";
```